docs/source/changelog.rst

```diff
@@ -38,6 +38,7 @@ Changelog
         * Implement automated process for checking critical dependencies (:pr:`1045`, :pr:`1054`, :pr:`1081`)
         * Don't run changelog check for release PRs or automated dependency PRs (:pr:`1057`)
         * Fix non-deterministic behavior in Dask test causing codecov issues (:pr:`1070`)
+        * Use CircleCI matrix jobs in config to trigger multiple runs of same job with different parameters (:pr:`1105`)
```

Comment:
move up to future release section and uncomment future release line

Reply:
Fixed